packages/react-examples/src/react/PeoplePicker/docs/PeoplePickerBestPractices.md

```diff
@@ -2,3 +2,11 @@
 
 - Use the people picker to add someone to the To line of an email, or to add someone to a list.
 - Use the `MemberList PeoplePicker` to display selections below the input field.
+
+## Accessibility
+
+PeoplePicker dropdowns render in their own layer by default to ensure they are not clipped by containers with `overflow: hidden` or `overflow: scroll`. This causes extra difficulty for people who use touch-based screen readers, so we recommend rendering the PeoplePicker inline unless it is in an overflow container. To do so, set the following property on the PeoplePicker:
+
+```js
+pickerCalloutProps={{ doNotLayer: true }}
+```
```

Comment:
Super nitpicky/opinion-based but I think it looks a bit funny to have the prop on its own line (up to you though).
![image](https://user-images.githubusercontent.com/5864305/121426051-fa966580-c927-11eb-8b6a-43aa1c382edb.png)

```suggestion
PeoplePicker dropdowns render in their own layer by default to ensure they are not clipped by containers with `overflow: hidden` or `overflow: scroll`. This causes extra difficulty for people who use touch-based screen readers, so we recommend rendering the PeoplePicker inline unless it is in an overflow container. To render inline, set `pickerCalloutProps={{ doNotLayer: true }}` on the PeoplePicker.
```

Reply:
This is the only comment I didn't change -- I'm leaning towards keeping it on a new line, just because I'm really hoping people will actually use it, so I'd like to make it visually called out & easy to copy/paste. I'm not super tied to it though -- in vNext, I think I'd actually like it to be inline by default, with a prop to render it in a new layer 😄